File: 260126/100점 도달하기/reach-100-point.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // Please write your code here.
        Scanner sc=new Scanner(System.in);

        int grade=sc.nextInt();

        for(int i=grade; i<101; i++){
            if(i>89){
                System.out.printf("A ");
            }
            else if(i>79){
                System.out.printf("B ");
            }
            else if(i>69){
                System.out.printf("C ");
            }
            else if(i>59){
                System.out.printf("D ");
            }
            else{
                System.out.printf("F ");
            }
        }
    }
}

File: 260224/100점 도달하기/reach-100-point.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int N;
    cin>>N;

    while(N<101){
        N>=60?
            (N>=70?
                (N>=80?
                    (N>=90?cout<<"A": cout<<"B")
                :cout<<"C")
            :cout<<"D")
        :cout<<"F";

        cout<<" ";
        N++;
    }
    return 0;
}